packages/app/jest.config.js

```diff
@@ -17,7 +17,7 @@ const config = {
   preset: 'ts-jest/presets/js-with-ts',
   resetMocks: true,
   roots: ['<rootDir>/src'],
-  setupFilesAfterEnv: ['<rootDir>/src/setupTests.js'],
+  setupFilesAfterEnv: ['<rootDir>/src/setupTests.ts'],
```

Comment:
I had some typing issues in tests and thought this might fix it: https://github.com/testing-library/jest-dom#usage. Turns out I had to install the `@types` package explicitly to ensure TypeScript could find it. Since converting the file to TS caught some useful issues, I decided to keep the change.